pymess/config.py

```diff
@@ -121,23 +95,31 @@
 }
 
 
+class BackendNotFound(Exception):
+    pass
+
+
 class Settings(object):
     """
     Pymess settings is loaded lazy like Django settings.
     The reason is usability of override_settings decorator with tests
     """
 
     def __getattr__(self, attr):
-        if attr not in DEFAULTS:
-            raise AttributeError('Invalid Pymess setting: "{}"'.format(attr))
-
-        default_value = DEFAULTS[attr]
-        value = getattr(django_settings, 'PYMESS_{}'.format(attr), DEFAULTS[attr])
-
-        if isinstance(default_value, dict) and isinstance(value, dict):
-            default_value = default_value.copy()
-            default_value.update(value)
-            value = AttrDict(default_value)
+        try:
+            default_value = DEFAULTS[attr]
+        except KeyError:
+            if re.search(r'_\d+$', attr):
```

Comment:
You don't need this anymore, or?

Reply:
it's leftover from the previous change. It's not needed now. Removed. Thanks.